composer.json

```diff
@@ -7,23 +7,23 @@
     "require": {
         "php": ">=7.1",
         "silverstripe/recipe-plugin": "^1",
-        "cwp/cwp-installer": "2.x-dev",
-        "cwp/agency-extensions": "2.x-dev",
-        "cwp/watea-theme": "3.x-dev",
-        "silverstripe/controllerpolicy": "2.x-dev",
-        "silverstripe/crontask": "2.x-dev",
-        "silverstripe/gridfieldqueuedexport": "2.x-dev",
-        "silverstripe/ldap": "1.x-dev",
-        "silverstripe/recipe-content-blocks": "2.x-dev",
-        "dnadesign/silverstripe-elemental-userforms": "3.x-dev",
-        "silverstripe/textextraction": "3.x-dev",
-        "silverstripe/realme": "4.x-dev",
-        "silverstripe/ckan-registry": "1.x-dev",
-        "silverstripe/webauthn-authenticator": "4.x-dev",
-        "silverstripe/security-extensions": "4.x-dev",
-        "silverstripe/login-forms": "4.x-dev",
-        "symbiote/silverstripe-multivaluefield": "5.x-dev",
-        "silverstripe/subsites": "2.x-dev"
+        "cwp/cwp-installer": "2.8.x-dev",
+        "cwp/agency-extensions": "2.5.x-dev",
+        "cwp/watea-theme": "3.1.x-dev",
+        "silverstripe/controllerpolicy": "2.2.x-dev",
+        "silverstripe/crontask": "2.2.x-dev",
+        "silverstripe/gridfieldqueuedexport": "2.4.x-dev",
+        "silverstripe/ldap": "1.2.x-dev",
+        "silverstripe/recipe-content-blocks": "2.8.x-dev",
+        "dnadesign/silverstripe-elemental-userforms": "3.0.x-dev",
+        "silverstripe/textextraction": "3.2.x-dev",
+        "silverstripe/realme": "4.0.x-dev",
```

Comment:
This should be 4.1.x-dev

See https://github.com/silverstripe/cwp-recipe-kitchen-sink/commit/0e8062607b464a33bb01f9afe1700b8da6b73438

Reply:
Do you mean `4.1.x-rc1`?

https://travis-ci.com/github/silverstripe/cwp-recipe-kitchen-sink/jobs/506501537

Reply:
er no :-) `4.1.x-dev` so that it used the tip of the `4.1` branch - just like all the other deps e.g. textextraction is on `3.2.x-dev`

Reply:
```
Root composer.json requires silverstripe/realme 4.1.x-dev, found silverstripe/realme[dev-master, dev-feature/2/ite-datacom, 0.9.0, 0.9.1, 2.0.0, 2.0.1, 2.0.x-dev, 2.1.0-rc1, 2.1.0-rc2, 2.1.0, 2.x-dev, 3.0.0-rc1, 3.0.0, 3.0.x-dev, 3.1.0, 3.1.1, 3.1.2, 3.1.3, 3.1.4, 3.1.x-dev, 3.x-dev, 4.0.0, 4.0.1, 4.0.2, 4.0.x-dev, 4.1.0-rc1, 4.1.0, 4.x-dev] but it does not match the constraint.
```

Reply:
Looks like the 4.1.0 branch was created off the `4` branch without first creating a `4.1` branch

I've created a new `4.1` branch off `4`, so this should work now

Reply:
I've re-ran travis - the failures look unrelated.